frontend/src/components/transactions/TransactionsHeader.jsx

```diff
@@ -1,30 +0,0 @@
-import { Row, Col } from "react-bootstrap";
```

Comment:
Those files were not used in the project.
Should we create an issue to have a PR check for unused modules?